namui/src/namui/system/wheel.rs

```diff
@@ -0,0 +1,34 @@
+use super::{platform_utils::web::window, InitResult};
+use crate::{
+    namui::{self, Xy},
+    RawWheelEvent,
+};
+use wasm_bindgen::{prelude::Closure, JsCast};
+
+pub async fn init() -> InitResult {
+    window()
+        .document()
+        .unwrap()
+        .add_event_listener_with_callback_and_add_event_listener_options(
+            "wheel",
+            Closure::wrap(Box::new(move |event: web_sys::WheelEvent| {
+                if event.ctrl_key() {
+                    event.prevent_default()
+                }
+
+                namui::event::send(namui::NamuiEvent::Wheel(RawWheelEvent {
+                    id: format!("wheel-{:?}-{:?}", namui::now(), namui::now()),
```

Comment:
`namui::now()` was called twice. Is it intended?

Reply:
Nope, I'm gonna fix it